docker/README.md

```diff
@@ -0,0 +1,25 @@
+# Docker
+This is the entry-point for getting the luigi system running with Docker.
+
+## Build Staging Scheduler Locally
+
+### Setup
+You will FIRST need to make a file named `client.cfg` and put it inside `/luigi/docker/staging-scheduler/etc/luigi/client.cfg`.
+
+```
+[scheduler]
+record_task_history: True
+state-path: /luigi/state/luigi-state.pickle
+
+[task_history]
+db_connection: sqlite:////luigi/state/luigi-task-history.db
+```
+
+### Run
+To run the staging scheduler locally, we must build it and then run it WHILE forwarding a PORT to open it on locally.
+
+```bash
+cd /luigi/docker/staging-scheduler
+docker build -t my-tag .
+bocker run -p 8082:8082 my-tag
+```
```

Comment:
I'm pretty new to Docker, what would an example of using a volume look like? 

This is something I don't fully understand yet. I agree with you; also I would say since we are unsure how we will deploy yet I think probably doing the docker volume approach could be better here (for now) to allow people to build/test/startup quickly.

Reply:
Definitely. One of the flags you can pass to `docker run` is `-v`, which is to mount a directory on your local filesystem into the container at a specified path. So, e.g. if I pass in `-v $HOME/data:/data`, the data in `$HOME/data` on my host machine will be available in the container at the path `/data`. Something similar can be accomplished with Kubernetes.

You can play with it a little bit, even just using the container you're building. In that case, you'll want to override the entrypoint, e.g. `docker run -it -p 8082:8082 --entrypoint /bin/bash -v whatever:whatever name-of-your-image`. The `-it` flag will make the session interactive (i.e. you can issue commands within the container), and `--entrypoint /bin/bash` will give you a shell.